tests/acceptance/15_control/02_agent/secureinput.cf

```diff
@@ -0,0 +1,31 @@
+body common control
+{
+  inputs => { "../../default.cf.sub" };
+  bundlesequence => { default("$(this.promise_filename)") };
+}
+
+body agent control
+{
+    secureinput => "true";
+}
+
+bundle agent init
+{
+  files:
+    "$(this.promise_dirname)/secureinput.cf.sub"
+    perms => bad_perms;
+      
+  commands:
+    "$(sys.cf_agent) --file $(this.promise_dirname)/secureinput.cf.sub";
+}
+
+body perms bad_perms
+{
+    owners => { "nobody" };
+}
+
+bundle agent check
+{
+  reports:
+    "$(this.promise_filename) Pass";
```

Comment:
Again, how could this fail?

Reply:
It seems this is a broken test. I will work on fixing it.